### DISCUSSION TITLE
Управление физганом для левшей

### DISCUSSION THREAD

**Discussion by Hesalix** [octothorp-team/public-dobrograd#719] [Предложения]
Здравствуйте, писал об этом предложении ещё 1,5 года назад. При взятии пропа на ЛКМ и попытке его повернуть на ENTER, захват пропа ЛКМом сбрасывается из-за открытия контекстного меню взаимодействия, SHIFT (правый) также не работает (однако захват пропа не сбрасывается).

**Comment by SnakeLuck** [comment]
За. Я как человек который переучивался с левой руки на правую могу сказать, это писец как бесит, когда ты не можешь нормально играть только из-за того что ты левша. Да это не распространенная проблема, но она очень весомая ткк это очень сильно портит игровой процесс

**Reply by Hesalix** [reply]
Плюсом ещё и при голоде нельзя зажимать шифт при стройке, не работает.
